code/__DEFINES/_math.dm

```diff
@@ -5,7 +5,7 @@
 #define PI 3.1415
 #define CARDINAL_DIRS 		list(1,2,4,8)
 #define CARDINAL_ALL_DIRS 	list(1,2,4,5,6,8,9,10)
-
+#define cheap_hypotenuse(Ax,Ay,Bx,By) sqrt(abs(Ax - Bx)**2 + abs(Ay - By)**2) //A squared + B squared = C squared
```

Comment:
The entire thing needs to be enclosed in parentheses.

Reply:
whoops!